src/libs/ajax.js

```diff
@@ -1110,9 +1110,23 @@ const DataRepo = signal => ({
       details: async () => {
         const res = await fetchDataRepo(`repository/v1/snapshots/${snapshotId}`, _.merge(authOpts(), { signal }))
         return res.json()
+      },
+      exportSnapshot: async () => {
+        const res = await fetchDataRepo(`repository/v1/snapshots/${snapshotId}/export`, _.merge(authOpts(), { signal }))
+        return res.json()
```

Comment:
For these: what happens if the call fails?  Do we need to have error handling if the export kick off fails?

Reply:
I also just ask in case the response isn't JSON (e.g. if a token has expired that can happen). That may be a fundamental change that is out of scope for the ticket...just wanted to check

Reply:
So, terra-ui has a generic error handler. I believe it won't finish the await, and will likely either cancel the execution of the method, or poll, that will error, and it will pop up the unsupported modal. I can test though.

Reply:
Added error handling for this and below!